LLMonFHIR/FHIR Display/MultipleResourcesChatView.swift

```diff
@@ -25,27 +24,30 @@ struct MultipleResourcesChatView: View {
     @AppStorage(StorageKeys.enableTextToSpeech) private var textToSpeech = StorageKeys.Defaults.enableTextToSpeech
     
     
-    private var disableInput: Binding<Bool> {
-        Binding(
-            get: {
-                multipleResourceInterpreter.viewState == .processing
-            },
-            set: { _ in }
-        )
-    }
-    
     var body: some View {
         @Bindable var multipleResourceInterpreter = multipleResourceInterpreter
         NavigationStack {
-            ChatView($multipleResourceInterpreter.chat, disableInput: disableInput)
+            Group {
+                if let llm = multipleResourceInterpreter.llm {
+                    let contextBinding = Binding { llm.context } set: { llm.context = $0 }
+                    ChatView(
+                        contextBinding,
+                        disableInput: multipleResourceInterpreter.viewState == .processing
+                    )
+                        .onChange(of: llm.context) {
+                            multipleResourceInterpreter.queryLLM()
+                        }
+                } else {
+                    ChatView(
+                        .constant([])
+                    )
+                }
```

Comment:
Same as in the documentation in Spezi LLM: The risk of showing an empty Chat view here that the user could interact with is not really ideal. Maybe there is a way that this could be solved in a reusable manner?